Sources/CoreUITestKit/XCUICoordinate+.swift

```diff
@@ -0,0 +1,12 @@
+
+import XCTest
+import Foundation
+
+extension XCUICoordinate {
+    public func dsl_press(forDuration: TimeInterval, thenDragTo: XCUICoordinate) {
```

Comment:
```suggestion
    public func dsl_press(forDuration duration: TimeInterval, thenDragTo dragTo: XCUICoordinate) {
```
так аргументы будут поприятнее в плане нейминга

Reply:
исправил, спасибо!